### DISCUSSION TITLE
Array undefined Context

### DISCUSSION THREAD

**Discussion by ivanjeremic** [vercel/next.js#14034] [Help] [answered]
My Context.Provider works in Nextjs plain JavaScript projects but the same project converted to TypeScript is not working the array I need from the context is in the TypeScript project undefined, anyone an idea why? In both projects, the provider is in the same place in _app.js/ts

**Comment by jamesmosier** [comment]
Could you share some code of it working and the TypeScript version that is not working?

**Reply by ivanjeremic** [reply]
That will take some time I need to prepare examples to not share the full project.

**Comment by jamesmosier** [comment, marked as answer]
The issue is with trying to access the context within your `react-konva` `Stage` wrapper in [`tools/ImageMapGenerator/MainStage.tsx`](https://github.com/ivanjeremic/webby.tools/blob/41f261b5d8c82302b3d89cc92aa477ddd48d21db/tools/ImageMapGenerator/MainStage.tsx#L39-L50). Once I removed `<Stage>` I was able to access the context values.

There is some more info [in their README](https://github.com/konvajs/react-konva#usage-with-react-context) on a work around.

**Reply by ivanjeremic** [reply]
Thank you, I missed this one.
